### PR TITLE
Make status bar prefix compact items a bit more compact

### PR DESCRIPTION
**Pull request by JasonInOttawa**
For battery level, show icon only
Reduce separator between items to " " from "  "

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7153)

### CODE REVIEW

**Comment by poire-z** [comment]
You probably did rebase to be sync'ed with our master - but in doing that, you restore bits from before https://github.com/koreader/koreader/commit/dffe86dfe9e3cb1d51687e1a4111eda6ae379b5a

**Comment by JasonInOttawa** [comment]
i didn't specifically do a rebase - I forked the koreader repository and made this change. Should I close this and perhaps resubmit the PR later?

**Comment by poire-z** [comment]
No, but you probably made this change to the file before, forked the repo and copy that file over the one from the forked updated repo.
You just need to get the latest readerfooter.lua and re-apply your changes on top of it.

File: frontend/apps/reader/modules/readerfooter.lua [hunk 1]
```diff
@@ -290,10 +294,11 @@ local footerTextGeneratorMap = {
         local prefix = symbol_prefix[symbol_type].mem_usage
         local statm = io.open("/proc/self/statm", "r")
         if statm then
-            local dummy, rss = statm:read("*number", "*number")
+            local infos = statm:read("*all")
             statm:close()
+            local rss = infos:match("^%S+ (%S+) ")
             -- we got the nb of 4Kb-pages used, that we convert to Mb
-            rss = math.floor(rss * 4096 / 1024 / 1024)
+            rss = math.floor(tonumber(rss) * 4096 / 1024 / 1024)
```
Comment (poire-z) [inline frontend/apps/reader/modules/readerfooter.lua:301]: You/we just don't want this part to appear in the diff :) it's not your work, just the glitch I'm talking about.

Comment (JasonInOttawa) [reply frontend/apps/reader/modules/readerfooter.lua]: That's exactly what I did - thank you!

**Review by poire-z — APPROVED** [review]

**Pull request merged**